test/e2e/docker/Dockerfile

```diff
@@ -1,13 +1,12 @@
 # We need to build in a Linux environment to support C libraries, e.g. RocksDB.
 # We use Debian instead of Alpine, so that we can use binary database packages
 # instead of spending time compiling them.
-FROM golang:1.23.5-bullseye
+FROM golang:1.23.6
 
 RUN apt-get -qq update -y && apt-get -qq upgrade -y >/dev/null
-RUN apt-get -qq install -y libleveldb-dev librocksdb-dev >/dev/null
 
 # Set up build directory /src/cometbft
-ENV COMETBFT_BUILD_OPTIONS badgerdb,boltdb,cleveldb,rocksdb
+# ENV COMETBFT_BUILD_OPTIONS badgerdb,boltdb,cleveldb,rocksdb
```

Comment:
nit: if not needed anymore, let's delete it

```suggestion
```

Reply:
makes sense

Reply:
this was applied to the new docker file for the e2e test